src/base/net/downloadmanager.h

```diff
@@ -164,6 +167,7 @@ namespace Net
         QSet<ServiceID> m_sequentialServices;
         QSet<ServiceID> m_busyServices;
         QHash<ServiceID, QQueue<DownloadHandlerImpl *>> m_waitingJobs;
+        QHash<ServiceID, std::chrono::seconds> m_serviceDelay;
```

Comment:
IMO, having yet another field is redundant. I would just modify existing one:
```c++
QHash<ServiceID, std::chrono::seconds> m_sequentialServices;
```

Reply:
Sounds good, i made those changes at 9eb1dc23062def333d7db4e16fb21decffa878ed